src/igvfd/schemas/changelogs/construct_library_set.md

```diff
@@ -5,6 +5,8 @@
 * Add a `title` property to `tile`.
 * Allow underscores in the pattern for `chromosome` in `loci`.
 * Add `release_timestamp`.
+* Add `small_scale_gene_list`, `large_scale_gene_list`, `small_scale_loci_list`, and `large_scale_loci_list`.
+* Restrict `genes` and `loci` to submittable by admins only. Please submit `small_scale_gene_list`, `large_scale_gene_list`, `small_scale_loci_list`, or `large_scale_loci_list` instead.
```

Comment:
remove the part "Please submit..." i dont think it is a part of change log
in general the idea is to have as concise as possible statements making it possible to read through changelog

Reply:
i think schema version also needs to be bumped

Reply:
removed